openedx/adg/common/lib/mandrill_client/client.py

```diff
@@ -28,6 +28,8 @@ class MandrillClient(object):
     APPLICATION_SUBMISSION_CONFIRMATION = 'adg-application-submission-confirmation-1'
     APPLICATION_WAITLISTED = 'adg-waitlisted-application'
     APPLICATION_ACCEPTED = 'adg-application-accepted'
+    WEBINAR_CREATED = 'webinar-invite'
+    WEBINAR_UPDATED = 'adg-webinar-update-email'
```

Comment:
would it be better to change the value of WEBINAR_CREATED to something that follows the convention of the values that the other constants hold? i.e. adg-webinar-invite-email